Core/Src/iwdg.c

```diff
@@ -36,13 +36,10 @@ void MX_IWDG_Init(void)
 
   /* USER CODE BEGIN IWDG_Init 1 */
 
-  // IWDG takes clock from LSE (32Khz)
-  // Elapse period ms = (IWDG_PRESCALER_4/LSE)*Reload = 10ms 
-
   /* USER CODE END IWDG_Init 1 */
   hiwdg.Instance = IWDG;
   hiwdg.Init.Prescaler = IWDG_PRESCALER_4;
-  hiwdg.Init.Reload = 80-1;
+  hiwdg.Init.Reload = 4095;
```

Comment:
```suggestion
  hiwdg.Init.Reload = 4096-1;
```
It's more clear what is the period tick